geometry_msgs/PoseStamped relative_pose
float64 target_velocity
---
int32  error_code
int32  GOAL_EXCEPTION=-1
string error_string
---
int8 state
int8 MOVE = 0
int8 FAIL = -1
int8 DONE = 1
